src/main/java/com/avioconsulting/mule/logger/internal/config/CustomLoggerConfiguration.java

```diff
@@ -312,6 +345,15 @@ public void start() throws MuleException {
     customLoggerRegistrationService.setConfig(this);
     if (isEnableFlowLogs()) {
       classLogger.info("Flow logs enabled");
+      flowLogConfigs.forEach(flowLogConfig -> {
+        if (flowLogConfig.getExpressionText().getMessageExpressionText() == null
+            && flowLogConfig.getExpressionText().getAttributesExpressionText() == null) {
+          throw new IllegalStateException(
+              "One of attributesExpressionText or messageExpressionText must be defined in flow-logs-config");
+        }
+      });
```

Comment:
We should move this validation to `initialise()` method where other validations are and the type of exception should be `InitialisationException`.

Reply:
moved to initialise() method